### ISSUE TITLE
Spiderfoot Scan Delete Failed - No Such File or Directory (2)

### ISSUE THREAD

**Issue by UncleRaymondo** [dtag-dev-sec/tpotce#190]
On attempting to delete a scan that has been running for quite some time, the following messages are received:

![image](https://user-images.githubusercontent.com/5147133/37442459-928a5594-285a-11e8-83c2-e0de0fb43457.png)

On clicking the "Click here to delete it" link, only a white page appears, however within /var/log/nginx/error.log, the following is found:

![spiderfoot_error](https://user-images.githubusercontent.com/5147133/37442506-dffc99cc-285a-11e8-8435-7c1ded388801.png)

Not too sure where to look at fixing this one just yet... Appreciate the assist...

Cheers,
Ray

**Comment by t3chn0m4g3** [comment]
This seems to be a related to #182. Please understand that like Kibana, rewriting rules are place. Changing ports usually should not affect them. Your screenshot shows that the rewrite rule does not match and tries to run `scandelete` from the local filesystem rather forwarding it to upstream.

**Comment by UncleRaymondo** [comment]
Hmm... Interesting...
I haven't changed any rewrite rules yet and this was run locally so using the internal IP and the correct port which has no NAT'ing on it...
Maybe I'll run the site through fiddler and see if I can find out why #182 is not working which may shed some answers on this...

**Comment by t3chn0m4g3** [comment]
Indeed, thanks for the heads up. Looking forward to some info to break it down.

**Comment by UncleRaymondo** [comment]
Hmm.. definitely a 404 per fiddler...

![image](https://user-images.githubusercontent.com/5147133/37511570-669dbf1c-2953-11e8-87f4-6c20be3c8c80.png)

Default Spiderfoot Rules in `tpotweb.conf`

![Uploading image.png…]()

Nothing really that I can find insider the spiderfoot container...
Interesting thing is I can browse through an entire scan of Spiderfoot, everything seems to work, it's just this delete command which is really weird...

**Comment by UncleRaymondo** [comment]
Ah! Found it!! 
Look very closely at the delete link... It's missing `/spiderfoot/` - so it's basically going `IP:64297/scandelete`

I added `/spiderfoot/` manually and it worked! 

![image](https://user-images.githubusercontent.com/5147133/37512122-4e33ae3a-2955-11e8-9665-8800951eb2ae.png)

Question now is... why... do we need to add a `proxy_pass` rule in `tpotweb.conf`??

**Comment by UncleRaymondo** [comment]
Looks like adding a proxy_pass rule in there may have fixed it...

![image](https://user-images.githubusercontent.com/5147133/37512679-ebe60546-2956-11e8-8edf-2df3588bd469.png)

![image](https://user-images.githubusercontent.com/5147133/37512703-fa23bee6-2956-11e8-84d8-00493c458c43.png)

**Comment by t3chn0m4g3** [comment]
Thanks. Will provide general fix for 18.04.

**Comment by UncleRaymondo** [comment]
Great, thanks buddy :)
